components/bpmn/org.wso2.carbon.bpmn.analytics.publisher/src/main/java/org/wso2/carbon/bpmn/analytics/publisher/AnalyticsPublisher.java

```diff
@@ -0,0 +1,303 @@
+/**
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.bpmn.analytics.publisher;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.bpmn.analytics.publisher.internal.BPMNAnalyticsHolder;
+import org.wso2.carbon.bpmn.analytics.publisher.models.BPMNProcessInstance;
+import org.wso2.carbon.bpmn.analytics.publisher.models.BPMNTaskInstance;
+import org.wso2.carbon.bpmn.analytics.publisher.utils.AnalyticsPublishServiceUtils;
+import org.wso2.carbon.bpmn.analytics.publisher.utils.BPMNDataReceiverConfig;
+import org.wso2.carbon.context.CarbonContext;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.context.RegistryType;
+import org.wso2.carbon.databridge.agent.thrift.DataPublisher;
+import org.wso2.carbon.databridge.agent.thrift.exception.AgentException;
+import org.wso2.carbon.databridge.commons.AttributeType;
+import org.wso2.carbon.databridge.commons.StreamDefinition;
+import org.wso2.carbon.databridge.commons.exception.*;
+import org.wso2.carbon.registry.api.Registry;
+import org.wso2.carbon.registry.api.RegistryException;
+import org.wso2.carbon.registry.core.utils.RegistryUtils;
+import org.wso2.carbon.user.api.UserStoreException;
+
+import java.net.MalformedURLException;
+import java.util.concurrent.Executors;
+
+/**
+ * AnalyticsPublisher uses to publish events to the data receiver in data-bridge
+ */
+public class AnalyticsPublisher {
+    private static final Log log = LogFactory.getLog(AnalyticsPublisher.class);
+
+    private String processInstanceStreamId;
+    private String taskInstanceStreamId;
+    private DataPublisher dataPublisher;
+    private AnalyticsPublishServiceUtils analyticsPublishServiceUtils;
+
+    /**
+     * Initialize the objects for AnalyticsPublisher
+     */
+    public void initialize() {
+        try {
+            RegistryUtils.setTrustStoreSystemProperties();
+            dataPublisher = createDataPublisher();
+            if(dataPublisher != null){
```

Comment:
Need to have a space after 'if'. Need to format all the places using wso2 code style